BlueChip/assetlist.json

```diff
@@ -0,0 +1,34 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "BlueChip",
+    "assets": [
+      {
+        "description": "The native staking token of BlueChip.",
+        "denom_units": [
+          {
+            "denom": "ubluechip",
+            "exponent": 0
+          },
+          {
+            "denom": "ebl",
+            "exponent": 6
+          }
+        ],
+        "base": "ubluechip",
+        "name": "blue chip",
+        "display": "bcp",
+        "symbol": "BCP",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/BlueChip/images/BlueChip.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/BlueChip/images/BlueChip.svg"
+        },
+        "coingecko_id": "",
+        "images": [
+          {
+            "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/BlueChip/images/BlueChip.png",
```

Comment:
please lowercase the directory name and file name in these urls (the uploaded file name is 'bluechip.png', not BlueChip.png)